static/src/javascripts/bootstraps/enhanced/atoms.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+
+import config from 'lib/config';
+import { services } from 'journalism/services';
+
+const bootstrapAtom = (atomMaker: AtomMaker, atomType: AtomType) => {
+    const atomBuilder = atomMaker[atomType].default(services);
+    [
+        ...document.querySelectorAll(`[data-atom-type='${atomType}']`),
+    ].forEach(atomDom => {
+        const atom = atomBuilder(atomDom).runTry();
+        if (typeof atom === 'string') {
+            // eslint-disable-next-line no-console
+            console.log(
+                `Failed to initialise atom [${atomType}/${atomDom.getAttribute(
+                    'data-atom-id'
+                ) || ''}]: ${atom}`
+            );
+        } else {
+            atom.start();
+        }
+    });
+};
+
+const initAtoms = () => {
+    if (config.get('page.atomTypes.guide')) {
+        require.ensure(
```

Comment:
why `require.ensure` rather than `import`?

Reply:
@SiAdcock would this work?

Reply:
Yeah, `import` will work. We only went with `require.ensure` for consistency. Happy to change it out

Reply:
I haven't been able to get this to work with `import()` due to the inability with this syntax to specify dependencies to the module (i.e. the CSS)